docs/application/dotnet/guides/nui/customview.md

```diff
@@ -82,9 +82,133 @@ To create a custom view:
     }
     ```
 
-    The following table lists other important custom view methods that you can use to manage the view.
+5. Define the constructor to create an instance of the custom view. In this case, the `ContactView` is created with the `contactName` parameter, and the resource directory path points to the icons. Additionally, the `base` constructor has to be called with the type name and view behavior as described in the subsequent section. In a nonstatic constructor, `Visuals` used to create a custom view should be created. For instance, in the following `ContactView` example, one label, background, and a few icons are shown:
 
-    **Table: Custom view methods**
+    ![Contact View](media/contact.png)
+
+    ```
+    public ContactView(string contactName, string resDir) : base(typeof(ContactView).Name, CustomViewBehaviour.ViewBehaviourDefault)
+        {
+            CreateBackground(new Vector4(1.0f, 1.0f, 1.0f, 1.0f));
+            CreateIcon(resDir + "/images/cbg.png", 10.0f, 5.0f, 100.0f, 100.0f, CONTACT_BG_ICON_INDEX);
+            CreateIcon(resDir + "/images/contact.png", 10.0f, 5.0f, 100.0f, 100.0f, CONTACT_ICON_INDEX);
+            CreateIcon(resDir + "/images/edit.png", 130.0f, 40.0f, 50.0f, 50.0f, CONTACT_EDIT_INDEX);
+            CreateIcon(resDir + "/images/favorite.png", 180.0f, 40.0f, 50.0f, 50.0f, CONTACT_FAVORITE_INDEX);
+            CreateIcon(resDir + "/images/delete.png", 640.0f, 40.0f, 50.0f, 50.0f, CONTACT_DELETE_INDEX);
+            CreateLabel(name);
+        }
+    ```
+
+6. Implementation of `Visuals` used in the `ContactView` example is shown in the following example:
+
+    > [!IMPORTANT] 
```

Comment:
@nawab-samsung, @mijin85cho 
Here is my opinion:
Important and recommended do not go along.
If it is important then it is unavoidable or critical. If it is recommended then it is a suggestion.

So we could change this to a Note as per me as it is recommendation and not a mandatory step.

Reply:
@annie-samsung 

@mihashco @mijin85cho : Kindly let me know your opinion regarding the same, you can refer to the discussion here: 

https://github.com/Samsung/tizen-docs/pull/1344#discussion_r524873982